src/frontend/typer/utbetalingAndel.ts

```diff
@@ -22,8 +22,8 @@ export enum IEndretUtbetalingAndelÅrsak {
 
 export const årsakTekst: { [key in IEndretUtbetalingAndelÅrsak]: string } = {
     DELT_BOSTED: 'Delt bosted',
-    ENDRE_MOTTAKER: 'Endre mottaker, begge foreldre rett',
-    ALLEREDE_UTBETALT: 'Allerede utbetalt',
+    ENDRE_MOTTAKER: 'Endre mottaker, begge foreldre rett. Opphør',
+    ALLEREDE_UTBETALT: 'Allerede utbetalt. Innvilge',
```

Comment:
Dette er en faglig avklaring isåfall, men hadde det vært finere med 
```
Endre mottaker, begge foreldrene rett (opphør)
Allerede utbetalt (innvilge)
```
eller
```
Endre mottaker, begge foreldrene rett - opphør
Allerede utbetalt - innvilge
```
?